package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "fh-mbaas-api",
-  "version": "5.13.9",
+  "version": "5.13.11",
```

Comment:
@matzew I released 5.13.10 as well but then made another change to fh-db and had to bump the version again. Sorry, forgot to tag it. will do.